### ISSUE TITLE
ctags are required if per-language confuguration specifies another binary

### ISSUE THREAD

**Issue by nponeccop** [preservim/tagbar#94]
I'm using `lushtags` for haskell and have  `let g:tagbar_type_haskell = { 'ctagsbin' : 'lushtags', ... }` configured.

However, I still need `ctags` executable in addition to `lushtags` as Tagbar refuses to start without it.

**Comment by d0u9** [comment]
Is there any update on this issue? I encounter this problem when using Golang and gotags. It seems that ctag is compulsory to boot tarbar successfully.

**Comment by alerque** [comment]
No, there is no update. I don't even know if @majutsushi left the incomplete work in abstracting the external calls and flag management anywhere it can be reviewed.

I'd be happy to facilitate the merger of any contribution that takes care of this cleanly. That being said I'm unlikely to have the time to work on in any time soon, especially since it isn't a problem I personally run into.

As another hack you could probably fix your system by adding a phony `ctags` somewhere in your path:

```console
$ ln -s /usr/bin/true /usr/bin/local/ctags
```

I haven't actually tested it, but if that doesn't work a small shell script that just prints out a ctags-ish version string might work as a shim.
